src/App/App.tsx

```diff
@@ -96,7 +95,6 @@ export default function App() {
             path={appRoutes.stripe_payment_status}
             element={<StripePaymentStatus />}
           />
-          <Route path={appRoutes.leaderboard} element={<Leaderboard />} />
```

Comment:
also need to remove `appRoutes.leaderboard` from `src/constants/routes.ts`

Reply:
Fixed in 6a3ff045.